components/CompareIndicators/BannerAside.tsx

```diff
@@ -0,0 +1,10 @@
+import React, { FunctionComponent } from 'react';
+
+interface BannerAsideProps {
+  classname: string;
+}
+
+const BannerAside: FunctionComponent<BannerAsideProps> = ({ children, classname }) => {
+  return <div className={classname}>{children}</div>;
+};
+export { BannerAside };
```

Comment:
This entire component is not useful. Plus it's a duplicate.

Reply:
Deleted this component